lib/Selenium/CanStartBinary.pm

```diff
@@ -121,6 +121,20 @@ has '+port' => (
     }
 );
 
+has 'marionette_port' => (
+    is => 'lazy',
+    builder => sub {
+        my ($self) = @_;
+
+        if ($self->isa('Selenium::Firefox') && $self->marionette_enabled) {
```

Comment:
off the cuff: does it make sense/is it possible to move this logic in to the `Selenium::Firefox` class itself to avoid the `$self->isa` check entirely? seems _slightly_ cleaner